components/SpeakersSection.js

```diff
@@ -41,6 +41,22 @@ export default function TicketsSection () {
           display: block;
           margin: 50px 0 0;
         }
+
+        @media only screen and (min-width: 66.25em) {
```

Comment:
The `em`-width is not resolved until the web font loads. Therefore I recommend using px units which makes the layout more stable.